packages/menu/menu-item.md

```diff
@@ -57,7 +57,33 @@ Content assigned to the `icon` slot will be placed at the beginning of the `<sp-
 </sp-menu>
 ```
 
-### Description slot
+##### Icon management
+
+When you use `<sp-menu-item>` elements without text content, you will need to be sure to use the `value` attribute so that the `<sp-picker>` or `<sp-menu>` element can differentiate between the available options. Further, it is important that you apply accessible labeling to the `[slot="icon"]` content as follows:
+
+```html
+<sp-field-label for="picker-icons-only">Choose an action...</sp-field-label>
+<sp-picker
+    label="What would you like to do?"
+    value="item-2"
+    id="picker-icons-only"
+>
+    <sp-menu-item value="item-1">
+        <sp-icon-save-floppy slot="icon" label="Save"></sp-icon-save-floppy>
+    </sp-menu-item>
+    <sp-menu-item value="item-2">
+        <sp-icon-stopwatch slot="icon" label="Finish"></sp-icon-stopwatch>
+    </sp-menu-item>
+    <sp-menu-item value="item-3">
+        <sp-icon-user-activity
+            slot="icon"
+            label="Review"
+        ></sp-icon-user-activity>
+    </sp-menu-item>
+</sp-picker>
+```
+
+#### Description
 
 Content assigned to the `description` slot will be placed below the `<sp-menu-item>`, like help text for users to understand the context of corresponding `<sp-menu-item>`.
```

Comment:
```suggestion
Content assigned to the `"description"` slot, eg help text, will be placed below the `<sp-menu-item>` to help users understand the context of corresponding `<sp-menu-item>`.
```

I'm reading this sentence as "content (like help text, for example) assigned to the slot helps users understand the context of the menu item" and put my suggestion accordingly. But if you meant "help text relating to a menu component is an example of how one could use the description slot", my suggestion probably would change here.